File: Python/Grace.py
#!/usr/bin/env python3

# My little Grace

NAME = "Grace_kid.py"
FILE = open(NAME, "w")
GRACE = '#!/usr/bin/env python3\n\n# My little Grace\n\nNAME = "Grace_kid.py"\nFILE = open(NAME, "w")\nGRACE = {0}\n\nFILE.write(GRACE.format(repr(GRACE)))\nFILE.close()\n'

FILE.write(GRACE.format(repr(GRACE)))
FILE.close()

File: JavaScript/Grace.js
#!/usr/bin/env node

/*
	My little Grace
*/

const FS = require("fs");
const NAME = "Grace_kid.js";
code = () => { return '#!/usr/bin/env node\n\n/*\n\tMy little Grace\n*/\n\nconst FS = require("fs");\nconst NAME = "Grace_kid.js";\ncode = ' + code.toString() + ' \n\nFS.writeFileSync(NAME, code());\n' } 

FS.writeFileSync(NAME, code());
